create_json.hpp

```diff
@@ -0,0 +1,58 @@
+
+///////////////////////////////////////////////////////////////////////////////
+// create_json.hpp
+//
+// Creates JSON file for unit tests output
+//
+// Author: Namanh Tran (namanhtran@csu.fullerton.edu)
+// Last Modified: Apirl 1, 2020
+///////////////////////////////////////////////////////////////////////////////
+
+#include <boost/property_tree/ptree.hpp>
+
+using boost::property_tree::ptree;
+
+// Creates JSON file for unit test outputs
+class UnitTestJSON {
```

Comment:
This project is using snake_case variable names to match the C++ standard library. Please change the class name to unit_test_json.

Reply:
All changes has been made!